src/Reflection/ReflectionProperty.php

```diff
@@ -12,30 +12,34 @@
  */
 class ReflectionProperty
 {
-    const IS_PRIVATE   = \ReflectionProperty::IS_PRIVATE;
-    const IS_PROTECTED = \ReflectionProperty::IS_PROTECTED;
-    const IS_PUBLIC    = \ReflectionProperty::IS_PUBLIC;
-    const IS_STATIC    = \ReflectionProperty::IS_STATIC;
+    private $modifiers;
 
-    private $modifiers   = null;
-    private $doc_comment = null;
-    private $default     = null;
-    private $class       = null;
-    private $name        = '';
+    /**
+     * @var string|null
```

Comment:
shouldn't it just be `?string` in this docblock too?

Reply:
`?string` is not valid as a docblock. It has to be `string|null` :cry: